### PR TITLE
k8s: use iptables for firewall 

### PR DESCRIPTION
**Pull request by mb-wali**

### CODE REVIEW

File: firewalld-config.yml [hunk 1]
```diff
@@ -10,26 +10,6 @@
   roles:
     - firewalld-config
```
Comment (mb-wali) [inline firewalld-config.yml:11]:
```suggestion
    - iptables-config
```

Comment (mb-wali) [reply firewalld-config.yml:11]: this would also need to rename the role directory.

**Review by mb-wali — COMMENTED** [review]
looks good, i would change the playbook name from `firewalld-config` to `iptables-config`

**Comment by tufranz** [comment]
playbook and role renamed. Readme updated (and a lot of comments removed 🙂).

File: roles/iptables-config/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# - name: Change input policy to DROP
+#   ansible.builtin.replace:
+#     path: /etc/iptables/rules.v4
+#     regexp: '(\*filter\n)(:INPUT) ACCEPT'
+#     replace: '\1\2 DROP'
+#   tags:
+#     - firewall
+#   notify:
+#     # - Save firewall-rules
+#     - Reload firewall-rules
```
Comment (mb-wali) [inline roles/iptables-config/tasks/main.yml:65]: remove the commented line - if not used

**Review by mb-wali — COMMENTED** [review]
cleanup - the commented codes

File: prepare.yml
Comment (mb-wali) [inline prepare.yml:1]: add some docs - in README - about this playbook/role -

File: roles/iptables-config/handlers/main.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+
+# - name: Save firewall-rules
+#   ansible.builtin.shell:
+#     cmd: "/usr/sbin/netfilter-persistent save"
+#     # creates: '/etc/iptables/rules.v4'
+
+- name: Reload firewall-rules
+  ansible.builtin.systemd:
+    name: iptables
```
Comment (mb-wali) [inline roles/iptables-config/handlers/main.yml:10]: shouldnt here reload nftables if debian12

Comment (tufranz) [reply roles/iptables-config/handlers/main.yml:5]: Handler is called only with CentOS7

**Pull request merged**